requirements/requirements.txt

```diff
@@ -6,3 +6,4 @@ numpy>=1.14.5
 GPy[plotting]>=1.10.0
 emcee>=2.2.1
 scipy>=1.1.0
+torch>=1.5.1
```

Comment:
Dependencies for examples are not added to requirements intentionally. We are lucky enough to have lots of examples in Emukit, and they carry different dependencies, non of which are required by the core lib. So the approach in Emukit is to only make users install core dependencies.

For running examples users are supposed to receive clear import errors, and manage pip install themselves.